### ISSUE TITLE
Modifier and special keys

### ISSUE THREAD

**Issue by aerickt** [Anodynous/stenogotchi#2]
Are there any plans to support modifier keys for shortcuts like ctrl + c and ctrl + v? I've also noticed that arrow keys and function keys don't seem to work.

**Comment by Anodynous** [comment]
Yes I intend to extend the support for modifier/function keys. Excluding media keys this is already working in QWERTY mode. Besides common modifiers needed for text output these are however not properly handled when received from Plover yet.

**Comment by aerickt** [comment]
Glad to hear that. Thank you for your fantastic work.

**Comment by Anodynous** [comment]
4d22b9240eba83e96822074a260ad3f717816cd3 adds support for modifier, navigation and function keys along with some other previously missing ones. Support for key combinations, like ctrl+alt+del and ctrl+c, has also been added. Let me know if you notice any issues.

Most accented letters, international characters and special symbols still aren't supported. This is mainly due to limitations of the Stenogotchi presenting itself as a bluetooth keyboard. A solution leveraging Alt/Option-codes could be implemented, but would depend on knowing the OS of the remote device to produce the right sequence. Media keys are still not working.

**Comment by aerickt** [comment]
So far I don't see any problems and it's working as intended. Some media keys are working (play, stop, and previous). I'm going to be using the Stenogotchi a lot more now, so I'll update if I run into anything.
